templates/netlify/remix.init/edge-server.js

```diff
@@ -1,6 +1,9 @@
 // Import path interpreted by the Remix compiler
 import * as build from "@remix-run/dev/server-build";
 import { createRequestHandler } from "@remix-run/netlify-edge";
+
+globalThis.process ||= { env: Deno.env.toObject() };
```

Comment:
Rather than putting this in the template, how about putting it in the `server.ts` in the `@remix-run/netlify-edge` package?